.github/workflows/main.yml

```
@@ -407,6 +407,20 @@ jobs:
         name: dist_linux_aarch64_gnu
         path: "target/aarch64-unknown-linux-gnu/release/wasm*"
 
+  dist_linux_aarch64_musl:
+    runs-on: ubuntu-24.04-arm
+    steps:
+    - uses: actions/checkout@v5
+    - run: rustup update --no-self-update stable && rustup default stable
+    - run: rustup target add aarch64-unknown-linux-musl
+    - run: sudo apt update -y && sudo apt install musl-tools -y
+    - run: |
+        cargo build --manifest-path crates/cli/Cargo.toml --target aarch64-unknown-linux-musl --features vendored-openssl --release
```

Apparently we are stripping debug symbols from x86 MUSL for Linux as well ...
```suggestion
    - env:
        RUSTFLAGS: -C strip=debuginfo
      run: |
        cargo build --manifest-path crates/cli/Cargo.toml --target aarch64-unknown-linux-musl --features vendored-openssl --release
```